packages/puppeteer-core/package.json

```diff
@@ -129,13 +129,7 @@
     }
   },
   "files": [
-    "lib",
```

Comment:
Revert this change this was reported back that we should include src so that source maps work.
Also the other prevents test from being in the submitted upload.

Reply:
It's okay to remove this now. The source map will point back to the GitHub url.